app/src/main/java/org/simple/clinic/contactpatient/ContactPatientUiRenderer.kt

```diff
@@ -78,6 +78,13 @@ class ContactPatientUiRenderer(
       ui.hidePatientWithNoPhoneNumberUi()
       ui.setResultOfCallLabelText()
 
+      loadSecureCallingUi(model)
+    } else if (model.patientProfileHasPhoneNumber && model.hasLoadedAppointment) {
```

Comment:
This `else if` clause seems to be identical to the parent `if` clause. Please take a look?

Reply:
Never mind: resolved in a later commit.